jedi/inference/compiled/access.py

```diff
@@ -487,7 +487,10 @@ def needs_type_completions(self):
     def _annotation_to_str(self, annotation):
         if isinstance(annotation, type):
             return str(annotation.__name__)
-        return str(annotation)
+        s = str(annotation)
+        if getattr(annotation, '__module__', None) == 'typing':
```

Comment:
Can you add an `and s.startswith('typing.')` and add a `1` as a param to replace? That way it will not break in the future.

Reply:
Done